tests/testthat/test-argufy_package.R

```diff
@@ -0,0 +1,59 @@
+{
+
+  library(disposables)
+
+  pkgs <- make_packages(
```

Comment:
Maybe I would put this in a function and call it from the test cases that need it. And also call `dispose_packages` at the end of both test cases.